documentation/misc/RELEASE_ISSUE_TEMPLATE.md

```diff
@@ -57,21 +55,24 @@
    - Link to PR:
    - There likely aren't any changes at this point, but this can be opened with a small whitespace change so the PR is open and we can more easily hold the standard of making improvements incrementally since improvements are usually better done by collecting changes/notes along the way rather than just thinking about it at the end.
    - This will get merged in a `Post Release` step.
-- [ ] (patch release) Fork a new branch (`release/vX.Y.Z` or `release/miner/vX.Y.Z`) from the last stable `release/vX.Y.Z` or `release/miner/vX.Y.Z` and make any further release-related changes to this branch.
+- [ ] (patch release) Fork a new branch (`release/vX.Y.Z` or `release/miner/vX.Y.Z`) from the last stable `release/vX.Y.x` or `release/miner/vX.Y.x` and make any further release-related changes to this branch.
 - [ ] (minor release) Fork a new branch (`release/vX.Y.Z` or `release/miner/vX.Y.Z`) from `master` and make any further release-related changes to this branch.
-- (minor release) `master` branch Version string updates
+- `master` branch Version string updates
+   - Skip this set of steps if you are patching a previos minor release. 
    - [ ] bump the version(s) in `build/version.go` to `vX.Y.(Z+1)-dev`. 
       - Ensure to update the appropriate version string based on whether you are creating a node release (`NodeBuildVersion`), a miner release (`MinerBuildVersion`), or both.
    - [ ] Run `make gen && make docsgen-cli` before committing changes.
-   - [ ] Remove the contents in the `Unreleased Changelog` section.
-   - [ ] Create a PR with title `build: update Lotus Node|Miner version to vX.Y.Z+1-dev in master`
+   - [ ] Update the CHANGELOG
+     - [ ] Change the `UNRELEASED` section to `UNRELEASED vX.Y.Z` and set the section's content to be `See https://github.com/filecoin-project/lotus/blob/release/vX.Y.Z/CHANGELOG.md`
+     - [ ] Add a new `UNRELEASED` header to top.
```

Comment:
@rjan: I made these changes so there is a pointer to where the UNRELEASED items are (i.e., there in a release branch now)